Update Readme.md (#1)
DevExpress-Examples/asp-net-web-forms-dropdownedit-template-with-treelist: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E2854)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

> # DropDown Edit for ASP.NET Web Forms - How to implement ASPxTreeList inside window template

Reviewer: ```suggestion
# DropDown Edit for ASP.NET Web Forms - How to implement ASPxTreeList inside a window template
```

<!-- run online -->
**[[Run Online]](https://codecentral.devexpress.com/e2854/)**
<!-- run online end -->

> This example demonstrates how to embed [ASPxTreeList](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxTreeList.ASPxTreeList) into [ASPxDropDownEdit](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxDropDownEdit)'s dropdown window template. Click on a node within the [ASPxTreeList](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxTreeList.ASPxTreeList) changes the editor's value.

Reviewer: Something is missing in the last sentence.

Author: ```suggestion
This example demonstrates how to embed [ASPxTreeList](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxTreeList.ASPxTreeList) into [ASPxDropDownEdit](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxDropDownEdit)'s dropdown window template. When a user clicks a node within ASPxTreeList, the value of ASPxDropDownEdit changes.
```
